packages/react-core/src/components/NumberInput/examples/NumberUnit.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { NumberInput } from '@patternfly/react-core';
+
+export const UnitNumberInput: React.FunctionComponent = () => {
+  const [value1, setValue1] = React.useState(90);
+  const [value2, setValue2] = React.useState(Number((1.0).toFixed(2)));
```

Comment:
So I'm noticing here that the `1.00` which was part of the original example is now displaying at `1` - and it's actually because by passing `(1.0).toFixed(2)` to a `Number` constructor, you are casting a string to a number and losing the extra trailing two zeros. 

This is necessary because the type of the `value` prop on `NumberInput` is `number | null`, so it cannot accept a string `1.00` value. It's possible we could update the number input value to accept strings and cast them to numbers internally. Not sure what side effects there'd be for that.

@tlabaj I'm not sure if you have thoughts about how we can handle this. I can open a follow up issue if we want - only if we are happy to the update to the example

Reply:
I think getting rid of `Number` here will fix the issue.

Reply:
I would say it is fine as is then.  The previous implementation was technically incorrect since it had `value2: (1.0).toFixed(2)`